src/main/java/com/google/growpod/data/User.java

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.data;
+
+/** User data class. */
+public class User {
+
+  public static final int MAX_BIOGRAPHY_LEN = 160;
+
+  /** A unique id. */
+  private String id;
+
+  private String email;
+  /** This is the only name we track. */
+  private String preferredName;
+
+  private String biography;
+  /**
+   * We do not wish to keep exact user locations -- this will store some approximate location data.
+   * Unfortunately, this will only work in the US.
```

Comment:
This is an interesting restriction and consideration for how y'all may want to handle garden location. Does this limit the region wherein people can find gardens, or also the region wherein people can register gardens?

Reply:
Can a user be an admin of a garden outside their zip code?

Reply:
The field `User.zipCode` serves two purposes:

1) The city name associated with the zip code is displayed on the user's profile.
2) When a user searches for nearby gardens, this location is used.

Users can both join and administer gardens anywhere -- a garden's location is kept within the `Garden.lat` and `Garden.lng` fields.